CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## Version 5.3.0 - 2023-12-12
```

Comment:
Will probably need to update the version after code review as I think this one is already taken